server/tests/clients/test_google.py

```diff
@@ -0,0 +1,126 @@
+"""
+Test the google client module.
+
+Find discovery paths for http mocking the build queries at:
+https://discovery.googleapis.com/discovery/v1/apis
+"""
+import json
+import os
+from pathlib import Path
+from typing import Optional, Tuple
+
+from googleapiclient.http import HttpMockSequence
+from pytest import MonkeyPatch
+from src.clients.google import get_user_info, is_google_token_valid
+from src.utils.error import AppError, ErrorCode
+
+
+def patch_is_google_token_valid(
+    retAudience: str, retExpiresIn: int, retUserId: str
+) -> Tuple[Optional[AppError], bool]:
+    """Mock responses for is google token valid."""
+    discovery_mock = Path(
+        os.path.join(os.path.dirname(__file__), "oauth2-discovery.json")
+    ).read_text()
+
+    http = HttpMockSequence([({"status": "200"}, discovery_mock)])
```

Comment:
for some godforsaken reason, HttpMockSEQUENCE refuses to sequence and always returns the first one